spp_farmer_registry_demo/models/farmer.py

```diff
@@ -9,7 +9,7 @@ class Farmer(models.Model):
             ("married_monogamous", "Married Monogamous"),
             ("married_polygamous", "Married Polygamous"),
         ]
-    )
+    , allow_filter=True)
```

Comment:
@shashikala1998 since you're adding allow_filter in the fields of `spp_farmer_registry_demo` you need to add `spp_custom_filter` in the manifest file of this `spp_farmer_registry_demo` module.